key_files/totalinvtalk.yml

```diff
@@ -0,0 +1,17 @@
+---
+name            : TotalInvtAlk
+units           : Peq
+model           : MEDUSA
+modelgrid       : eORCA1
+dimensions      : 1
+modelFiles      : $BASEDIR_MODEL/$JOBID/medusa*_$JOBIDo_1y_*_diad-T.nc
+model_vars      : INVTALK
+model_convert   :
+    path: bgcval2/functions/InvtAlk.py
```

Comment:
I think you need to include this file in the review as well.

Reply:
Added now.